cloudsync_onedrive.py

```diff
@@ -1160,7 +1146,7 @@ def test_instance(cls):
 
     @property
     def _test_namespace(self):
-        return "personal"
+        return self.__drive_to_name[self._personal_id]
```

Comment:
this requires you to be connected first in order to know what namespace to connect to.   better to return a magic string "!test!" or something.

Reply:
This needs to be a real namespace because it is used in integration tests that make real API calls. Decided to revert this back to "personal" since that namespace maps to the user's personal drive and should always exist.